src/hotspot/share/gc/g1/g1FullGCCompactTask.cpp

```diff
@@ -74,6 +80,7 @@ size_t G1FullGCCompactTask::G1CompactRegionClosure::apply(oop obj) {
   cast_to_oop(destination)->init_mark();
   assert(cast_to_oop(destination)->klass() != NULL, "should have a class");
 
+  clear_bitmap(obj);
```

Comment:
Maybe a simple comment here will be helpful, although we already have comments where  is used.

Reply:
Added.